<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="4" total="6" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="388" started-at="2018-12-21T12:35:46Z" finished-at="2018-12-21T12:35:46Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="388" started-at="2018-12-21T12:35:46Z" finished-at="2018-12-21T12:35:46Z">
      <class name="com.ofs.training.java.test.PersonServletTest">
        <test-method status="PASS" signature="init()[pri:0, instance:com.ofs.training.java.test.PersonServletTest@1b604f19]" name="init" is-config="true" duration-ms="25" started-at="2018-12-21T12:35:46Z" finished-at="2018-12-21T12:35:46Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method status="FAIL" signature="testCreate(com.ofs.training.java.pojo.PojoServicePerson, com.ofs.training.java.pojo.PojoServicePerson, java.util.List)[pri:1, instance:com.ofs.training.java.test.PersonServletTest@1b604f19]" name="testCreate" duration-ms="311" started-at="2018-12-21T12:35:46Z" data-provider="dpCreate" finished-at="2018-12-21T12:35:46Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[PojoServicePerson [id=0, first_name=harish, last_name=kumar, email=harish@gmail.in, address_id=0, birth_date=14-08-1995, created_date=null, address=PojoServiceAddress [id=0, street=deras, city=chennai, postal_code=600095]]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[PojoServicePerson [id=4, first_name=harish, last_name=kumar, email=harish@gmail.in, address_id=7, birth_date=14-08-1995, created_date=null, address=PojoServiceAddress [id=21, street=deras, city=chennai, postal_code=600095]]]]>
              </value>
            </param>
            <param index="2">
              <value is-null="true"/>
            </param>
          </params>
          <exception class="java.lang.NoClassDefFoundError">
            <message>
              <![CDATA[org/apache/commons/logging/LogFactory]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NoClassDefFoundError: org/apache/commons/logging/LogFactory
at org.apache.http.conn.ssl.DefaultHostnameVerifier.<init>(DefaultHostnameVerifier.java:72)
at org.apache.http.impl.client.HttpClientBuilder.build(HttpClientBuilder.java:955)
at com.ofs.training.java.test.RequestHelper.requestRaw(RequestHelper.java:87)
at com.ofs.training.java.test.RequestHelper.requestEntity(RequestHelper.java:79)
at com.ofs.training.java.test.RequestHelper.requestObject(RequestHelper.java:131)
at com.ofs.training.java.test.RequestHelper.requestObject(RequestHelper.java:124)
at com.ofs.training.java.test.PersonServletTest.testCreate(PersonServletTest.java:35)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.ClassNotFoundException: org.apache.commons.logging.LogFactory
at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:349)
at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
... 32 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NoClassDefFoundError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCreate -->
        <test-method status="FAIL" signature="testCreate(com.ofs.training.java.pojo.PojoServicePerson, com.ofs.training.java.pojo.PojoServicePerson, java.util.List)[pri:1, instance:com.ofs.training.java.test.PersonServletTest@1b604f19]" name="testCreate" duration-ms="12" started-at="2018-12-21T12:35:46Z" data-provider="dpCreate" finished-at="2018-12-21T12:35:46Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[PojoServicePerson [id=0, first_name=viki, last_name=jai, email=vicky@gmail.in, address_id=0, birth_date=1997-07-04, created_date=null, address=PojoServiceAddress [id=0, street=  , city=, postal_code=0]]]]>
              </value>
            </param>
            <param index="1">
              <value is-null="true"/>
            </param>
            <param index="2">
              <value>
                <![CDATA[[STREET_NULL, CITY_NULL, POSTALCODE_NULL, NAME_DUPLICATED, EMAIL_DUPLICATED, DATE_FORMAT_INVALID]]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NoClassDefFoundError">
            <message>
              <![CDATA[org/apache/commons/logging/LogFactory]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NoClassDefFoundError: org/apache/commons/logging/LogFactory
at org.apache.http.conn.ssl.DefaultHostnameVerifier.<init>(DefaultHostnameVerifier.java:72)
at org.apache.http.impl.client.HttpClientBuilder.build(HttpClientBuilder.java:955)
at com.ofs.training.java.test.RequestHelper.requestRaw(RequestHelper.java:87)
at com.ofs.training.java.test.RequestHelper.requestEntity(RequestHelper.java:79)
at com.ofs.training.java.test.RequestHelper.requestObject(RequestHelper.java:131)
at com.ofs.training.java.test.RequestHelper.requestObject(RequestHelper.java:124)
at com.ofs.training.java.test.PersonServletTest.testCreate(PersonServletTest.java:35)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NoClassDefFoundError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCreate -->
      </class> <!-- com.ofs.training.java.test.PersonServletTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
